### ISSUE TITLE
Build failing because of deprecation

### ISSUE THREAD

**Issue by shntnu** [0x00B1/persistence#107]
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly

e.g. 

```
______________________ ERROR at setup of test_seed[htqc] _______________________
Fixture "htqc" called directly. Fixtures are not meant to be called directly,
but are created automatically when test functions request them as parameters.
See https://docs.pytest.org/en/latest/fixture.html for more information about fixtures, and
https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly about how to update your code.
```

**Comment by gwaybio** [comment]
it also looks like the `cytominer-database` build is failing because of an unmaintained dependency `odo` (see blaze/odo#619)''

This issue is also described in #69 and is the result of `odo` using `pandas.tslib`.

The future of this project requires us to seek an alternative to `odo`!

**Comment by gwaybio** [comment]
a quick search comes up with [d6tstack](https://github.com/d6t/d6tstack), but I will continue searching for alternatives

**Comment by gwaybio** [comment]
see #108 - working around `odo` with `pandas` and `sqlalchemy` seems to be a promising route

**Comment by gwaybio** [comment]
> https://docs.pytest.org/en/latest/deprecations.html#calling-fixtures-directly

fixed in 4501cad161ba3a6c027892c506bb049c35bbe4f5 🎉
